File: BibliaApp.Api/Biblia/Models/bible_verses.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Biblia.Models
{
    public class bible_verses
    {
        [Key]
        public int idVerse { get; set; }
        public int idBible { get; set; }
        public int idBook { get; set; }
        public int chapter { get; set; }
        public int verse { get; set; }
        public string text { get; set; }
    }
}

File: BibliaApp.Movil/app/src/main/java/com/rperdomo/bibliaapp/Lecturas/Model/bible_verses.java
package com.rperdomo.bibliaapp.Lecturas.Model;

public class bible_verses {

    private int idVerse;
    private int idBible;
    private int idBook;
    private int chapter;
    private int verse;
    private String text;

    public int getIdVerse() {
        return idVerse;
    }

    public void setIdVerse(int idVerse) {
        this.idVerse = idVerse;
    }

    public int getIdBible() {
        return idBible;
    }

    public void setIdBible(int idBible) {
        this.idBible = idBible;
    }

    public int getIdBook() {
        return idBook;
    }

    public void setIdBook(int idBook) {
        this.idBook = idBook;
    }

    public int getChapter() {
        return chapter;
    }

    public void setChapter(int chapter) {
        this.chapter = chapter;
    }

    public int getVerse() {
        return verse;
    }

    public void setVerse(int verse) {
        this.verse = verse;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

}
